### ISSUE TITLE
Equivalent of "model show" in PySPEX

### ISSUE THREAD

**Issue by drogantini** [spex-xray/spex-help#31]
Hi Jelle, 

quick question. What is the equivalent of "model show" in PySPEX?

Moreover, I would find it quite useful to have a model class with attributes like the number of free parameters, a list of all components, and total flux/luminosity. Something similar to the class [Model](https://heasarc.gsfc.nasa.gov/xanadu/xspec/python/html/model.html) in PyXspec. I am aware that the two Python interfaces are built differently, so it might not be easy/practical to do so. 

Thanks, 
Daniele

**Comment by jdeplaa** [comment]
Hi Daniele,

Thanks for the suggestion! Indeed, model show functionality is not yet there. The model module in pyspex does contain the data that you are looking for, but it is not yet accessible through an easy function. You probably mean something like the `par_show()` command, but then have a python readable table or list as output. I can put it on the wish list.

Cheers, Jelle.

**Comment by drogantini** [comment]
Yes indeed, it would be pretty useful to easily access the model information such as the number of free parameters, the name of the different model components, etc... A Python readable table would definitively help. Many thanks

**Comment by jdeplaa** [comment]
Hi Daniele,

I have created a couple of functions to get the information of par_show into multiple Astropy tables for the next spex version. For example, with the ``s.par_show_param()`` command you can get a table with all the parameters and their properties:
```
    >>> s.com('reds')
    >>> table = s.par_show_param()
    ---------------------------------------------------------------------------------------------------
    sect comp mod  acro parameter with unit      value     status    minimum   maximum lsec lcom lpar

       1    1 reds z    Redshift                 0.00000   frozen       -1.0   1.0E+10
       1    1 reds flag Flag: cosmo=0, vel=1     0.00000   frozen        0.0       1.0

    >>> print(table)
    Sect Comp Model Acro         Par          Value AValue Status         Min              Max      Link_sect Link_comp Link_par
    ---- ---- ----- ---- -------------------- ----- ------ ------ ------------------- ------------- --------- --------- --------
       1    1  reds    z             Redshift   0.0        frozen -0.9990000128746033 10000000000.0         0         0        0
       1    1  reds flag Flag: cosmo=0, vel=1   0.0        frozen                 0.0           1.0         0         0        0
```
The example above is just for a small reds component, but the table will contain the parameters of all loaded components.

In addition, I have also similar functions to get the free parameters only, to get a list of coupled parameters and a table with fluxes. Is this what you mean? Comments and suggestions welcome!

Cheers, Jelle.
